web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -601,73 +603,55 @@ private void parseCancer(Gene gene, Set<Alteration> alterations, JSONObject canc
         for (int i = 0; i < implications.length(); i++) {
             JSONObject implication = implications.getJSONObject(i);
             if ((implication.has("description") && !implication.getString("description").trim().isEmpty()) || (implication.has("treatments") && implication.getJSONArray("treatments").length() > 0)) {
```

Comment:
Since we don't have general descriptions, I can modify the condition to only call `parseTherapeuticImplications` if it has treatments.